pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -941,6 +941,13 @@ int socket__http2_eos_parser(struct __sk_buff *skb) {
             continue;
         }
 
+        // It is not possible that path will be with length 0
+        if ((current_stream->path.length == 0) && (current_stream->path.static_table_entry != kEmptyPath)) {
```

Comment:
you're missing kEmptyPath (how did it work in the CI then?)
you should check `!current_stream->path.finalized`

wondering if you should be running that after handle_end_of_stream

Reply:
Regarding the change after the handle_end_of_stream I do not think there is a reason to even try and handle end of stream for a request which we have with size 0, there is no meaning for that